### ISSUE TITLE
Attacking

### ISSUE THREAD

**Issue by AwesomestCode** [PrismarineJS/prismarine-web-client#9]
Attacking players

**Comment by ShrimpyStuff** [comment]
Yeah I can work on that if I can know that an entity is in front of me unless block breaking uses the same packet

**Comment by Picoseconds** [comment]
It doesn't

**Comment by ShrimpyStuff** [comment]
Ok cool

**Comment by Darkflame72** [comment]
Is there any progress on this or should I take it up? @ShrimpyStuff

**Comment by ShrimpyStuff** [comment]
You can take it up because I have no idea how to do it

**Comment by Darkflame72** [comment]
No problem will do

**Comment by ShrimpyStuff** [comment]
@Darkflame72 Do you have any progress?

**Comment by Darkflame72** [comment]
> @Darkflame72 Do you have any progress?

Should have a working version in the next few days as I wanted to fix some weird bugs it has

**Comment by Darkflame72** [comment]
Been super busy so haven't had it as a priority

**Comment by ShrimpyStuff** [comment]
Ok that's fine

**Comment by Darkflame72** [comment]
@ShrimpyStuff I am not going to have any time to work on this as have been super busy with uni and have a lot going on at the moment sorry

**Comment by ShrimpyStuff** [comment]
Yeah ok that's fine

**Comment by u9g** [comment]
A checklist for things to finish this:

- [ ] make bot.entityAtCursor() ( Mineflayer )
- [ ] implement it in P-web-client
